### PR TITLE
MM-15528: Updates to channel info for group-constrained channels.

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary

For group-constrained channels:
* Adds a line of text to the channel info view.
* Hides the "Add Member" link.
* Never shows "Manage Members", always "View Members".

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-15528
and
https://mattermost.atlassian.net/browse/MM-15529

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information

iPhone X, iOS 12.2 emulator.

#### Screenshots

Simulator Screen Shot - iPhone X - 2019-05-10 at 19 56 43
Simulator Screen Shot - iPhone X - 2019-05-10 at 19 57 33

### CODE REVIEW

File: app/screens/channel_members/index.js [hunk 1]
```diff
@@ -16,15 +16,17 @@ function makeMapStateToProps() {
...
-        const currentChannelId = getCurrentChannelId(state);
+        const currentChannel = getCurrentChannel(state);
         let currentChannelMembers = [];
-        if (currentChannelId) {
-            currentChannelMembers = getChannelMembers(state, currentChannelId, true);
+        if (currentChannel.id) {
+            currentChannelMembers = getChannelMembers(state, currentChannel.id, true);
         }
 
+        const canManageUsers = currentChannel.group_constrained ? false : canManageChannelMembers(state);
```
Comment (hmhealey) [inline app/screens/channel_members/index.js]: Minor nit, but I think this might be a bit clearer than using a ternary statement
```suggestion
        const canManageUsers = canManageChannelMembers(state) && !currentChannel.group_constrained;
```

**Comment by mkraft** [comment]
@hmhealey In the webapp you can manage the members by changing their role between `[team|channel]_user` and `[team|channel]_admin`. Furthermore, depending on QA capacity, with the release of this feature users may be able to be individually added to teams and channels if they're in the subset of system users who are members of associated groups. However that adding of users may be pushed to the next iteration. 

Put simply, `canManageChannelMembers` will need to be a separate boolean from `isGroupConstrained`.

**Comment by hmhealey** [comment]
Cool, I was just missing something then. That makes sense
